eng/performance/benchmark_jobs.yml

```diff
@@ -98,22 +98,13 @@ jobs:
             ${{ each framework in parameters.frameworks }}:
               ${{ framework }}:
                 _Framework: ${{ framework }}
-                _CompilationMode: NoTiering
-                _Configs: CompilationMode=$(_CompilationMode)
+                _Configs: CompilationMode=Tiered
```

Comment:
Did you mean `Default` here?

Reply:
see comment in the PR conversation.

Reply:
I guess a better question is: What's the configuration generated by the scripts after changing the command line arguments below? I think that removing `--dotnet-compilation-mode` will generate a new trace because the new compilation mode is `Default`.
Maybe I'm missing something here, but I believe the changes will generate a trace that is: `.NET Compilation Mode=[Default]`

Reply:
Ah, good point. I forgot that we didn't update what goes into the Benchview scripts yet to use the new variable. Let me think about this - maybe this is where we just pull the plug on Benchview. (It's happening in a couple weeks anyway.)

Reply:
If that's the case, we should remove all BenchView related scripts/commands/logic (it would be a good cleanup).

Reply:
> maybe this is where we just pull the plug on Benchview. (It's happening in a couple weeks anyway.)

We are now very close to the release date and I am using BenchView daily to validate that our fixes actually fix perf problems.

Could you please pull the plug after the 3.0 release or after we switch to the new reporting system?

Reply:
Of course. We'll transition smoothly, don't worry.

Reply:
For now I'm just going to hard code the value for benchview as well - I want to get to turning off the extra run we no longer need, and want to get to where we're collecting a baseline set of data without any `COMPlus` variables set. Removing upload to benchview can be another step.